src/card.ts

```diff
@@ -86,6 +86,7 @@ import {
   getOverriddenConfig,
   getOverridesByKey,
 } from './card-condition.js';
+import { supportsFeature } from './icons/update.js';
```

Comment:
Nice re-use. Would you mind moving this out of `icons/update.js` and into `common.js` since it's now used by more than just the icon transplant I had to do?

Reply:
This was the first thing I did, but then rollup:

```
(!) Circular dependency
src/common.ts -> src/icons/state-icon.ts -> src/icons/domain-icon.ts -> src/icons/update.ts -> src/common.ts
```

Reply:
Oh, dang. OK, leave it where it is. I'll refactor it a different way, probably pull the entity specific stuff out.